client/app/src/js/controllers/landing.js

```diff
@@ -4,11 +4,27 @@
  * Handles display landing page.
  */
 
-function LandingController() {
+function LandingController(Screenshots, Auth) {
+  this.data = {};
 
+  this.getScreenshots = function(){
+    // gets public screenshots
+    Screenshots.grabScreenshots()
+    .success(function(data){
+      this.results = data;
```

Comment:
Quite helpful: provide a sample `data` result in a comment. E.g. `// e.g. data = {username: 'andrew', ...}`